src/Definition/Helper/CreateDefinitionHelper.php

```diff
@@ -68,7 +68,7 @@ public function lazy() : self
      * This method takes a variable number of arguments, example:
      *     ->constructor($param1, $param2, $param3)
      *
-     * @param mixed[] $parameters Parameters to use for calling the constructor of the class.
+     * @param mixed $parameters Parameters to use for calling the constructor of the class.
```

Comment:
I think it should be like this for [variadic functions](https://phpstan.org/writing-php-code/phpdocs-basics#variadic-functions):

```suggestion
     * @param mixed ...$parameters Parameters to use for calling the constructor of the class.
```

Reply:
Let me confirm that that works, and I'll update it accordingly.